decidim-core/app/views/decidim/shared/_orders.html.erb

```diff
@@ -1,9 +1,10 @@
 <div class="row collapse order-by">
   <div class="order-by__dropdown">
     <span class="order-by__text"><%= t("#{i18n_scope}.label") %></span>
-    <ul class="dropdown menu" data-dropdown-menu data-close-on-click-inside="false">
-      <li>
-        <%= order_link order, i18n_scope: i18n_scope %>
+    <ul class="dropdown menu" data-dropdown-menu>
```

Comment:
Yeah @deivid-rodriguez, I'd remove this and tackle this problem in a separate PR if you don't mind :)